Octokit/Helpers/ApiUrls.cs

```diff
@@ -85,10 +86,19 @@ public static Uri Organizations()
         /// </summary>
         /// <param name="login">The login for the user</param>
         /// <returns></returns>
-        public static Uri Organizations(string login)
+        public static Uri UserOrganizations(string login)
         {
             return "users/{0}/orgs".FormatUri(login);
         }
+        
+        /// <summary>
+        /// Returns the <see cref="Uri"/> that returns all of the organizations for the currently logged in user.
```

Comment:
Extra text here from cut+paste :grinning:

Returns the <see cref="Uri"/> that returns all of the organizations ~~for the currently logged in user~~